### ISSUE TITLE
FFmpeg-converted video not showing properly in some of the standard players

### ISSUE THREAD

**Issue by eunjongkim** [JuliaGL/GLVisualize.jl#252]
Hi,

I noticed that the `*.mp4` videos I have created using `GLVisualize.jl` doesn't seem to show properly in some of the standard video players like Quicktime Player or Windows Media Player. It works with VLC though. 

I'm guessing this is related to the following issue: [link to stackoverflow](https://stackoverflow.com/questions/24693751/html5-video-ffmpeg-encoded-mp4-not-playing-in-any-browser-plays-in-vlc-though) 
I believe this could be solved by changing some of the options for ffmpeg encoding, which is used to create videos in the package.. Does anyone have thoughts on this ? Is there any way I can change the ffmpeg options for encoding the video?

Thanks in advance.

**Comment by SimonDanisch** [comment]
Yes! I figured that out as well and changed it in Makie: https://github.com/JuliaPlots/Makie.jl/blob/master/src/plotsbase/output.jl#L134 (the `-pix_fmt yuv420p` should be the important change).
How did you create an mp4 with GLVisualize? I just realize, that I only offer mkv & webm:
https://github.com/JuliaGL/GLVisualize.jl/blob/master/src/videotool.jl#L42
